### PR TITLE
More cypress tests

### PR DESCRIPTION
**Pull request by AndersonCeci**
Added more cypress tests 
I also added .pdf, .odt, and .docx file types to test the upload button
The text should be kept if clicking on the upload-document tab and then back on the writing tab.
Added test if  Opening and closing the off-canvas works as expected.
Added test if Clicking on the information button of some markings leads to pages with further details on the markings.

### CODE REVIEW

**Comment by AndiBraimllari** [comment]
Nice to see more tests being added, can you please split the tests into other files, not just inside `general-flow.cy.js`?

**Comment by AndersonCeci** [comment]
> Nice to see more tests being added, can you please split the tests into other files, not just inside `general-flow.cy.js`?

Say no more you got it

**Comment by AndiBraimllari** [comment]
Nice to see so many new files from the general-flow file, I'll have a look at the code once you're done.
Also just solved some merge conflicts.

**Comment by AndiBraimllari** [comment]
An issue has been created https://github.com/OpenCovenant/quill/issues/131 for the error in the workflow. I assume rerunning will give us green checks.

You can run `prettier:fix` to format the code.

Are you done with this PR?

File: cypress/e2e/x-button.cy.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+it('will click on the "x" button and check if the text is clear', () => {
+
+    cy.get("#editor > p > .typo").should("not.exist");
+    cy.get("#editor").type("test per butonin x");
+    cy.get('i.bi-x').click();
+    cy.get("#editor").contains("test per butonin x").should("not.exist");
+
+});
+
+});
```
Comment (AndiBraimllari) [inline cypress/e2e/x-button.cy.js:15]: Can you also test here that the delete button is not shown if there's no text in the editor? Perhaps you can check on line 7 and 12 whether it is shown.

**Comment by AndiBraimllari** [comment]
The changes from #133 might have solved the failing workflow, let's see if the error happens again.

**Comment by AndiBraimllari** [comment]
Do you plan to add more to this PR?

**Comment by AndersonCeci** [comment]
> Do you plan to add more to this PR?

Yes I will add  few more tests

**Comment by AndersonCeci** [comment]
Can you test on your end please to see if the copy button works properly because I had some kind of issues with it

**Comment by AndiBraimllari** [comment]
Hey, the pipeline seems to pass so I guess the test is working fine?
Regarding the created Command, could we inline it instead? As in the following
```
describe("going to test the copy button and paste it on editor", () => {
    beforeEach(() => {
        cy.visit("http://localhost:4200/");
    });

    it("will click on copy button and paste on editor", () => {
        cy.get("#editor").clear().type("test per butonin copy");

        cy.get("#copyToClipboardButton ").click();

        cy.window().then((win) => {
            win.navigator.clipboard.readText().then((text) => {
                expect(text).to.eq("test per butonin copy");
            });
        });
    });
});
```

**Comment by AndiBraimllari** [comment]
This PR should help quite a bit in regard to testing existing functionality.

**Review by AndiBraimllari — APPROVED** [review]

**Pull request merged**